web/src/components/graph/exchangelineargradients.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
```

Comment:
I was confused by the name but maybe we should call this exchangeseriesgradient? (to point out that it's the whole series and not just a list of exchanges)

Reply:
Sure, makes sense, the only thing is I'd like to keep it plural since it actually renders multiple gradients, so `ExchangeSeriesGradients`.